apps/worker/src/app/workflow/usecases/add-job/add-job.usecase.ts

```diff
@@ -275,36 +310,48 @@ export class AddJob {
       status: JobStatusEnum.QUEUED,
     });
 
-    await this.queueJob(job, 0);
+    await this.queueJob({ job, delay: 0, untilDate: null });
 
     return {
       workflowStatus: null,
       deliveryLifecycleStatus: null,
     };
   }
 
-  private async handleDelay(command: AddJobCommand, bridgeResponse: ExecuteOutput | null) {
+  private async handleDelay({
+    command,
+    job,
+    bridgeResponse,
+    bridgeDelayAmountDate,
+    bridgeDelayAmount,
+    timezone,
+  }: {
+    command: AddJobCommand;
+    job: JobEntity;
+    bridgeResponse: ExecuteOutput | null;
+    bridgeDelayAmountDate: Date | null;
+    bridgeDelayAmount: number | undefined;
+    timezone: string | undefined;
+  }) {
     let metadata: IWorkflowStepMetadata;
     if (bridgeResponse) {
       // Assign V2 metadata from Bridge response
-      metadata = await this.updateMetadata(bridgeResponse, command);
+      metadata = await this.updateMetadata(bridgeResponse, command, bridgeDelayAmountDate);
     } else {
       // Assign V1 metadata from known values
       metadata = command.job.step.metadata as IWorkflowStepMetadata;
     }
 
-    const delayAmount = await this.addDelayJob.execute(
```

Comment:
this usecase was hiding the logic but we didn't do the same for the digest, so I decided to simplify that and make both be more similar